setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -eu
+
+mkdir -p $INSTALL_DIR
```

Comment:
Should probably be quoted here and below to ensure directories with spaces work as expected. I've seen that somewhat regularly among people who are (or were) designers or QA, e.g. a "Git repositories" or "My Repositories" folder.

Reply:
Done.